client/luigi-client.d.ts

```diff
@@ -100,6 +101,21 @@ export declare interface UserSettings {
   [key: string]: number | string | boolean;
 }
 
+export declare interface CustomMessageData {
```

Comment:
is this **CustomMessageData** used anywhere. I cannot find any usage of it anywhere on the repo. Would be nice to aim to only make JS-TS transfer with the most minimal changes possible in this PR. Adding new data types would have to be analysed first or be made in another follow up PR.

Reply:
It seems this is some unnecessary leftover - to be removed